EndaMaryamDengelat/EDMG045.xml

```diff
@@ -0,0 +1,429 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="EDMG045" xml:lang="en" type="mss">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Fragments of various texts</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="DN"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+               <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+               <pubPlace>Hamburg</pubPlace>
+               <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                  Forschungsumgebung / Beta maṣāḥǝft</publisher>
+               <availability>
+                  <licence target="http://creativecommons.org/licenses/by-sa/4.0/">
+                     This file is licensed under the Creative Commons Attribution-ShareAlike 4.0.
+                  </licence>
+               </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS1017EndaMaryamDengelat"/>
+                        <idno>EDMG-045</idno>      
+                        <idno facs="EDMG/045/EDMG-045_" n="58">EDMG-045</idno>       
+                        </msIdentifier>
```

Comment:
please also check EDMG001 EDMG002 etc that have multiple `idno `elements - there should only be one, all others should be in `altIdentifier`

Reply:
Changed.